scripts/validate-cmux-browser-skill.py

```
@@ -0,0 +1,588 @@
+    errors: list[str] = []
+    for example in _logical_examples(examples):
+        if not example.text.strip() or example.text.lstrip().startswith("#"):
+            continue
+        try:
+            tokens = _tokenize(example.text)
+        except ValueError as exc:
+            errors.append(f"{example.path}:{example.line}: invalid shell syntax: {exc}")
+            continue
+
+        for index, token in enumerate(tokens):
+            if token != "cmux":
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "token"

(S105)